### PR TITLE
Fix schedule time

### PR DESCRIPTION
**Pull request by vdoleans01**

### CODE REVIEW

File: session-replacement/src/main/java/com/amadeus/session/SessionManager.java [hunk 1]
```diff
@@ -201,6 +201,8 @@ private RepositoryBackedSession fetchSession(String sessionId, boolean updateTim
       logger.debug("Session was in cache, but it was expired, sessionId: {}", sessionId);
 
       if (session.isValid()) {
+    	logger.info("deleting session with sessionId: '{}'", sessionId );
```
Comment (nbogojevic) [inline session-replacement/src/main/java/com/amadeus/session/SessionManager.java]: Formatting - missing space.

File: session-replacement/src/main/java/com/amadeus/session/repository/redis/NotificationExpirationManagement.java [hunk 2]
```diff
@@ -4,6 +4,7 @@
 import static java.util.concurrent.TimeUnit.MILLISECONDS;
 import static java.util.concurrent.TimeUnit.SECONDS;
 
+import java.text.SimpleDateFormat;
```
Comment (nbogojevic) [inline session-replacement/src/main/java/com/amadeus/session/repository/redis/NotificationExpirationManagement.java]: Is this used?

File: session-replacement/src/main/java/com/amadeus/session/repository/redis/NotificationExpirationManagement.java [hunk 3]
```diff
@@ -59,7 +60,7 @@
 class NotificationExpirationManagement implements RedisExpirationStrategy {
   static Logger logger = LoggerFactory.getLogger(NotificationExpirationManagement.class);
 
-  private static final long ONE_MINUTE = TimeUnit.MINUTES.toMillis(1);
+  private static final long ONE_MINUTE_IN_SECOND = TimeUnit.MINUTES.toSeconds(1);
```
Comment (nbogojevic) [inline session-replacement/src/main/java/com/amadeus/session/repository/redis/NotificationExpirationManagement.java]: Keep ONE_MINUTE as constant name.

File: session-replacement/src/main/java/com/amadeus/session/repository/redis/NotificationExpirationManagement.java [hunk 4]
```diff
@@ -147,7 +148,7 @@ public void sessionTouched(SessionData session) {
     @Override
     public void run() {
       long prevMin = roundDownMinute(System.currentTimeMillis());
-
+
```
Comment (nbogojevic) [inline session-replacement/src/main/java/com/amadeus/session/repository/redis/NotificationExpirationManagement.java]: Superfluous whitespace

File: session-replacement/src/main/java/com/amadeus/session/SessionManager.java [hunk 5]
```diff
@@ -616,4 +619,10 @@ public String toString() {
     return new StringBuilder().append("SessionManager [namespace=").append(configuration.getNamespace()).append("]")
         .toString();
   }
+
+  public void remove(SessionData sessionData) {
+	logger.info("deleting session with sessionId: '{}'", sessionData.getId() );
```
Comment (nbogojevic) [inline session-replacement/src/main/java/com/amadeus/session/SessionManager.java]: As we repeat 3 times 
```java
logger.info("deleting session with sessionId: '{}'", some-id);
deletedSessions.mark()
```
can you extract this into new markSessionDeletion method.

File: session-replacement/src/main/java/com/amadeus/session/SessionManager.java (hunk 1 above)
Comment (vdoleans01) [reply session-replacement/src/main/java/com/amadeus/session/SessionManager.java]: fixed

File: session-replacement/src/main/java/com/amadeus/session/SessionManager.java (hunk 5 above)
Comment (vdoleans01) [reply session-replacement/src/main/java/com/amadeus/session/SessionManager.java]: fixed

File: session-replacement/src/main/java/com/amadeus/session/repository/redis/NotificationExpirationManagement.java (hunk 2 above)
Comment (vdoleans01) [reply session-replacement/src/main/java/com/amadeus/session/repository/redis/NotificationExpirationManagement.java]: fixed

File: session-replacement/src/main/java/com/amadeus/session/repository/redis/NotificationExpirationManagement.java (hunk 3 above)
Comment (vdoleans01) [reply session-replacement/src/main/java/com/amadeus/session/repository/redis/NotificationExpirationManagement.java]: ok, if you want , but it takes me timje to unserstand that it is a minute in second units :-)

File: session-replacement/src/main/java/com/amadeus/session/repository/redis/NotificationExpirationManagement.java (hunk 4 above)
Comment (vdoleans01) [reply session-replacement/src/main/java/com/amadeus/session/repository/redis/NotificationExpirationManagement.java]: fixed

File: session-replacement/src/main/java/com/amadeus/session/SessionManager.java [hunk 6]
```diff
@@ -341,14 +340,19 @@ public void propagateSession(RequestWithSession request, ResponseWithSessionId r
...
-      deletedSessions.mark();
+      markSessionDeletion(sessionId);
       session.doInvalidate(expired);
     } else if (!expired) {
       logger.debug("Session not found in repository for sessionId: '{}'", sessionId);
     }
   }
 
+  private void markSessionDeletion(String sessionId) {
+	  logger.info("deleting session with sessionId: '{}'", sessionId );
```
Comment (nbogojevic) [inline session-replacement/src/main/java/com/amadeus/session/SessionManager.java]: Too much whitespace (2 space ident).

**Pull request merged**